src/WindowsTracing/EventTypes.h

```diff
@@ -0,0 +1,91 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef WINDOWS_TRACING_EVENT_TYPES_H_
+#define WINDOWS_TRACING_EVENT_TYPES_H_
+
+#include <cstdint>
+
+// https://docs.microsoft.com/en-us/windows/win32/etw/thread-typegroup1
+// [EventType{ 1, 2, 3, 4 }, EventTypeName{ "Start", "End", "DCStart", "DCEnd" }]
+struct Thread_TypeGroup1 {
```

Comment:
I'm a bit lost with this file, some explanatory comment might be necessary.
For example I'm confused by the structs. In the links they have fields but here you don't need them thanks to `krabs`? But then you use `struct`s of the same name as containers of the opcodes. I can see the point of using the same name but it's also confusing because then I expect the same exact `struct`, maybe a suffix would be useful? Or do you even need the structs if the opcodes are all distinct?

Reply:
Replaced `struct` by `namespace`. I like grouping them by event type better than having a flat set of opcodes. Also, added a comment.